starting_package/setup.py

```diff
@@ -1,6 +1,6 @@
 """
- OpenVINO Profiler
- Setup for the DL Workbench Python starter wrapper
+ OpenVINO DL Workbench Python Starter
```

Comment:
Are you sure that we have right headers? Take a look to the openvinotoolkit/model_analyzer